### PR TITLE
Fix not visible 0 value in editor and graph

### LINKED ISSUE #59: Zero number value is not showed
```js
await deep.select(1723)
{
    "data": [
        {
            "id": 1723,
            "type_id": 1606,
            "from_id": 0,
            "to_id": 0,
            "value": {
                "id": 7,
                "value": 0,
                "link_id": 1723
            },
            "__typename": "links"
        }
    ],
    "loading": false,
    "networkStatus": 7
}
```
image

In the editor it is not showed too
image

# How to reproduce
```ts
  const containTypeLinkId = await deep.id('@deep-foundation/core', 'Contain');
  const typeTypeLinkId= await deep.id('@deep-foundation/core', 'Type');
  const valueTypeLinkId= await deep.id('@deep-foundation/core', 'Value');
  const numberTypeLinkId= await deep.id('@deep-foundation/core', 'Number');

const {data: [{type_id}]} = await deep.insert({
      type_id: typeTypeLinkId,
      in: {
        data: {
          type_id: containTypeLinkId,
          from_id: deep.linkId,
          string: { data: { value: 'ActionSheetResultIndex' } },
        },
      },
      out: {
        data: {
          type_id: valueTypeLinkId,
          to_id: numberTypeLinkId
        }
      }
    })
    

  const actionSheetResultIndexTypeLinkId = await deep.id(PACKAGE_NAME, "ActionSheetResultIndex");
  console.log("Insert result", actionSheetResult.index)
  await deep.insert({
    type_id: containTypeLinkId,
    from_id: deep.linkId,
    to: {
      data: {
        type_id: type_id,
        number: {
          data: {
            value: 0
          }
        }
      }
    }
  })
```

### PR DESCRIPTION
**Pull request by FreePhoenix888**

### CODE REVIEW

**Comment by FreePhoenix888** [comment]
image

**Comment by FreePhoenix888** [comment]
70% to @FreePhoenix888 
30% to @flakeed

File: imports/cyto/editor.tsx [hunk 1]
```diff
@@ -281,7 +281,7 @@ export function CytoEditor() {
     });
   }, [tab?.initialValue, generatedLink]);
   
-  const currentValue = valuesRef?.current?.[tabId] || tab?.initialValue || '';
+  const currentValue = valuesRef?.current?.[tabId] || tab?.initialValue !== undefined ? tab?.initialValue : '';
```
Comment (Aruseli) [inline imports/cyto/editor.tsx]: when comparing against undefined , it's better to compare against undefined string using typeof
```js
typeof tab?.initialValue !== 'undefined'
```

Comment (FreePhoenix888) [reply imports/cyto/editor.tsx]: > when comparing against undefined , it's better to compare against undefined string using typeof
> 
> ```js
> typeof tab?.initialValue !== 'undefined'
> ```

Why?

**Comment by konard** [comment]
5000 rub reward is paid

**Pull request merged**